setup.py

```diff
@@ -23,16 +23,17 @@
     command_number = 'git rev-list --count HEAD'
 
     try:
-        commit_hash = subprocess.check_output(command, shell=True).decode('ascii').strip()
-        commit_number = subprocess.check_output(command, shell=True).decode('ascii').strip()
+        commit_hash = subprocess.check_output(command_hash, shell=True)\
+            .decode('ascii').strip()
+        commit_number = subprocess.check_output(command_number, shell=True)\
+            .decode('ascii').strip()
     except Exception:
         pass
     else:
-
         # We write the git hash and value so that they gets frozen if installed
         with open(os.path.join('glue', '_githash.py'), 'w') as f:
-            f.write("__githash__ = \"{githash}\"\n".format(githash=commit_hash))
-            f.write("__dev_value__ = \"{dev_value}\"\n".format(dev_value=commit_number))
+            f.write("__githash__ = \"{}\"\n".format(commit_hash))
+            f.write("__dev_value__ = \"{}\"\n".format(commit_number))
```

Comment:
These changes aren't really necessary - can you revert them? (also, ``{}`` won't work on Python 2.6)